.github/workflows/ci.yml

```
@@ -169,13 +169,14 @@ jobs:
   jdk-integration-tests:
     name: Java Distribution IT
     needs: build
-    runs-on: ubuntu-latest
     timeout-minutes: 100
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
         dist: [temurin]
         version: [17, 19]
```

No need to test both 17 and 19 on Windows. This should exclude 19 on Windows:

```suggestion
        version: [17, 19]
        exclude:
          - os: windows-latest
            version: 19
```